aissemble-open-inference-protocol-fastapi/src/aissemble_open_inference_protocol_fastapi/rest/endpoints.py

```diff
@@ -363,3 +441,28 @@ def build_inference_response(
         id=request.id,
         outputs=outputs,
     )
+
+
+def get_user_ip_from_request(request: Request):
+    """
+    This method extracts the user's IP address from the request object.
+    The x-forwarded-for header is used by proxy servers to preserve
+    the actual IP of the user, so if it's present we need to extract
+    the IP from it.
```

Comment:
A: Write for consumer, not implementer (also this is mainly just an excuse to use the new multi-line feature in Github 😄)
```suggestion
    Extracts the user's IP address from the request object, respecting the
    `x-forwarded-for` header if present.
```

Reply:
- [x] make code comment concise by using the suggested phrasing above